<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeScienze500</class>
 <widget class="QMainWindow" name="LeScienze500">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>687</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Le Scienze 500</string>
  </property>
  <property name="locale">
   <locale language="Italian" country="Italy"/>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frameParoleChiave">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>561</width>
      <height>71</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QCheckBox" name="Select_ParoleChiave">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>26</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ParoleChiave">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>251</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Parole chiave dal titolo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ParoleChiaveAbstract">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>261</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Parole Chiave dall'abstract</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CopiaADestra">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Copy &gt;&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CopiaASinistra">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;&lt; Copy</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="FrameRicercaEstesa">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>561</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ricerca Estesa nel testo</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Select_RicercaTesto">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="ParoleChiaveTesto">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>511</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameCategoria">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>261</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="ListaCategorie">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>241</width>
       <height>291</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="selectionRectVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="Select_RicercaCategoria">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Categorie</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CleanCategorie">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Clean</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="FrameAutori">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>221</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="ListaAutori">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>201</width>
       <height>261</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="Select_ListaAutori">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Autori</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="FiltroAutori">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="CleanAutori">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Clean</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="FrameAnno">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>81</width>
      <height>371</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="Select_Anno">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Anno</string>
     </property>
    </widget>
    <widget class="QListWidget" name="ListaAnni">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>301</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="CleanAnni">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Clean</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="AND_OR_Frame">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>520</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="AND_Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>AND</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="OR_Button">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>OR</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="FrameRubriche">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="Select_Rubriche">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>85</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Rubriche</string>
     </property>
    </widget>
    <widget class="QListWidget" name="ListaRubriche">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>261</width>
       <height>101</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="CleanRubriche">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Clean</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Cerca">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>570</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Cerca</string>
    </property>
   </widget>
   <widget class="QFrame" name="FrameRisultato">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>501</width>
      <height>661</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Risultato della ricerca</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="TabellaRisultati">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>481</width>
       <height>371</height>
      </rect>
     </property>
     <property name="columnCount">
      <number>0</number>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>481</width>
       <height>241</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="TabPrincipale">
      <attribute name="title">
       <string>Informazioni Articolo</string>
      </attribute>
      <widget class="QTextBrowser" name="MostraTitolo">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>471</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="MostraAbstract">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>471</width>
         <height>111</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="MostraData">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>170</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="MostraNumero">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>170</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ApriPDF_A">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>170</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Apri Articolo in PDF</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="TabAltro">
      <attribute name="title">
       <string>Altre Informazioni</string>
      </attribute>
      <widget class="QTextBrowser" name="MostraRubrica">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>291</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="MostraCategoria">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>291</width>
         <height>91</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="MostraFilePDF">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>40</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ApriPDF_B">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>140</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Apri Articolo in PDF</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="Configura">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>540</y>
      <width>99</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Configura</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Le Scienze 500</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1089</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
